pkg/landscaper/controllers/execution/controller.go

```diff
@@ -138,6 +142,34 @@ func HandleAnnotationsAndGeneration(ctx context.Context, log logr.Logger, c clie
 	return nil
 }
 
+// HandleDeployItemGeneration checks the managed DeployItems owned by this execution and will force a reconcilcation
+// by setting the Executions Phase to "Init" if an DeployItem cannot be retrieved or if a DeployItems's generation does not
```

Comment:
It says `Init` in the docstring, but the phase is actually set to `ExecutionPhaseProgressing`, which is `Progressing`